### PR TITLE
Paddle based ppo

### PR DESCRIPTION
**Pull request by ShuaibinLi**

### CODE REVIEW

File: examples/PPO/README.md [hunk 1]
```diff
@@ -7,22 +7,55 @@ Based on PARL, the PPO algorithm of deep reinforcement learning has been reprodu
 Please see [here](https://github.com/openai/mujoco-py) to know more about Mujoco games.
```
Comment (rical730) [inline examples/PPO/README.md]: Mujoco/Atari games introduction

File: examples/PPO/README.md [hunk 2]
```diff
@@ -7,22 +7,55 @@ Based on PARL, the PPO algorithm of deep reinforcement learning has been reprodu
...
 
++ Each experiment was run three times with different seeds
+
 ## How to use
 ### Dependencies:
 + [paddle>=2.0.0](https://github.com/PaddlePaddle/Paddle)
-+ [parl>=2.0.2](https://github.com/PaddlePaddle/PARL)
-+ gym==0.9.2
-+ mujoco-py==0.5.7
++ [parl>=2.0.5](https://github.com/PaddlePaddle/PARL)
```
Comment (rical730) [inline examples/PPO/README.md]: parl>2.0.5

File: examples/PPO/agent.py [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+        Args:
+            obs (np.array): observation
+        """
+        obs = paddle.to_tensor(obs, dtype='float32')
+        value, action, action_log_probs, action_entropy = self.alg.sample(obs)
+        return value.detach(
+        ).numpy(), action.detach().numpy()[0], action_log_probs.detach().numpy(
+        )[0], action_entropy.detach().numpy()
+
+    def value(self, obs):
```
Comment (rical730) [inline examples/PPO/agent.py:63]: please add doc for value, describe the shape of the input in this file

File: examples/PPO/README.md (hunk 1 above)
Comment (ShuaibinLi) [reply examples/PPO/README.md]: done

File: examples/PPO/README.md (hunk 2 above)
Comment (ShuaibinLi) [reply examples/PPO/README.md]: done

File: examples/PPO/agent.py (hunk 3 above)
Comment (ShuaibinLi) [reply examples/PPO/agent.py:63]: done

File: examples/PPO/atari_model.py [hunk 4]
```diff
@@ -18,6 +18,12 @@
 
 
 class AtariModel(parl.Model):
+    """ The Model for Mujoco env
```
Comment (rical730) [inline examples/PPO/atari_model.py]: Mujoco->Atari

File: parl/tests/gym.py [hunk 5]
```diff
@@ -142,7 +142,8 @@ def get_action_meanings(self):
                 return ['NOOP'] * 6
 
         self.observation_space = ObservationSpace((210, 160, 3), 'unit8')
-        self.action_space = ActionSpace(6)
+        self.action_space = ActionSpace(n=6, shape=())
```
Comment (rical730) [inline parl/tests/gym.py:156]: Is the shape of action sapce of HalfCheetah-v1 just ()?

File: examples/PPO/atari_model.py (hunk 4 above)
Comment (ShuaibinLi) [reply examples/PPO/atari_model.py]: done

File: parl/tests/gym.py (hunk 5 above)
Comment (ShuaibinLi) [reply parl/tests/gym.py:156]: add real obs/act space to mock env

**Review by rical730 — APPROVED** [review]
LGTM~

**Pull request merged**